benchmark/test_runner/run-single-test-file.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const { run } = require('node:test');
+const { writeFileSync, mkdirSync } = require('node:fs');
+const { join } = require('node:path');
+const tmpdir = require('../../test/common/tmpdir');
+const common = require('../common');
```

Comment:
Does this  need to be imported first in benchmarks the way it is in tests?

Reply:
Updated!